src/util/message.c

```diff
@@ -27,15 +27,20 @@
 #include <stdarg.h>
 #include <syslog.h>
 #include <libgen.h>
+#include <time.h>
 
 #include "config.h"
 #include "util/util.h"
 #include "util/message.h"
+#include "util/config_parser.h"
 
 int messagelevel = -1;
 
 extern const char *__progname;
 
+int log_file_descriptor;
+FILE *log_file;
```

Comment:
Do you need both the file descriptor as well as the file stream pointer? You should only require one (perhaps you want the file stream pointer as it is easier to obtain a FD from that?).